src/docs-map.ts

```diff
@@ -1,61 +1,61 @@
+  { name: 'Grid', dsd: 'grid', chakra: '/website/pages/docs/layout/grid.mdx' },
+  { name: 'SimpleGrid', dsd: 'simple-grid', chakra: '/website/pages/docs/layout/simple-grid.mdx' },
+  { name: 'Stack', dsd: 'stack', chakra: '/website/pages/docs/layout/stack.mdx' },
+  { name: 'Wrap', dsd: 'wrap', chakra: '/website/pages/docs/layout/wrap.mdx' },
 
-  { dsd: 'avatar', chakra: '/website/pages/docs/media-and-icons/avatar.mdx' },
-  { dsd: 'icon', chakra: '/website/pages/docs/media-and-icons/icon.mdx' },
-  { dsd: 'image', chakra: '/website/pages/docs/media-and-icons/image.mdx' },
+  { name: 'Avatar', dsd: 'avatar', chakra: '/website/pages/docs/media-and-icons/avatar.mdx' },
+  { name: 'Icon', dsd: 'icon', chakra: '/website/pages/docs/media-and-icons/icon.mdx' },
+  { name: 'Image', dsd: 'image', chakra: '/website/pages/docs/media-and-icons/image.mdx' },
 
-  { dsd: 'breadcrumb', chakra: '/website/pages/docs/navigation/breadcrumb.mdx' },
-  { dsd: 'link', chakra: '/website/pages/docs/navigation/link.mdx' },
-  { dsd: 'link-overlay', chakra: '/website/pages/docs/navigation/link-overlay.mdx' },
+  { name: 'Breadcrumb', dsd: 'breadcrumb', chakra: '/website/pages/docs/navigation/breadcrumb.mdx' },
+  { name: 'Link', dsd: 'link', chakra: '/website/pages/docs/navigation/link.mdx' },
+  { name: 'LinkOverlay', dsd: 'link-overlay', chakra: '/website/pages/docs/navigation/link-overlay.mdx' },
 
-  { dsd: 'accordion', chakra: '/website/pages/docs/disclosure/accordion.mdx' },
-  { dsd: 'tabs', chakra: '/website/pages/docs/disclosure/tabs.mdx' },
-  { dsd: 'visually-hidden', chakra: '/website/pages/docs/disclosure/visually-hidden.mdx' },
+  { name: 'Accordion', dsd: 'accordion', chakra: '/website/pages/docs/disclosure/accordion.mdx' },
+  { name: 'Tabs', dsd: 'tabs', chakra: '/website/pages/docs/disclosure/tabs.mdx' },
+  { name: 'VisuallyHidden', dsd: 'visually-hidden', chakra: '/website/pages/docs/disclosure/visually-hidden.mdx' },
 
-  { dsd: 'alert-dialog', chakra: '/website/pages/docs/overlay/alert-dialog.mdx' },
-  { dsd: 'drawer', chakra: '/website/pages/docs/overlay/drawer.mdx' },
-  { dsd: 'menu', chakra: '/website/pages/docs/overlay/menu.mdx' },
-  { dsd: 'modal', chakra: '/website/pages/docs/overlay/modal.mdx' },
-  { dsd: 'popover', chakra: '/website/pages/docs/overlay/popover.mdx' },
-  { dsd: 'tooltip', chakra: '/website/pages/docs/overlay/tooltip.mdx' },
-];
+  { name: 'AlertDialog', dsd: 'alert-dialog', chakra: '/website/pages/docs/overlay/alert-dialog.mdx' },
+  { name: 'Drawer', dsd: 'drawer', chakra: '/website/pages/docs/overlay/drawer.mdx' },
+  { name: 'Menu', dsd: 'menu', chakra: '/website/pages/docs/overlay/menu.mdx' },
+  { name: 'Modal', dsd: 'modal', chakra: '/website/pages/docs/overlay/modal.mdx' },
+  { name: 'Popover', dsd: 'popover', chakra: '/website/pages/docs/overlay/popover.mdx' },
+  { name: 'Tooltip', dsd: 'tooltip', chakra: '/website/pages/docs/overlay/tooltip.mdx' },
```

Comment:
This name can be computed using lodash's [camelCase](https://lodash.com/docs/4.17.15#camelCase) and [capitalize](https://lodash.com/docs/4.17.15#capitalize) functions. We don't need an extra prop in the map.

Reply:
done. removed the name property